src/Nethermind/Nethermind.Core.Test/Modules/MainBlockProcessingContext.cs

```diff
@@ -20,13 +21,14 @@ namespace Nethermind.Core.Test.Modules;
 /// <param name="GenesisLoader"></param>
 public record MainBlockProcessingContext(
     ILifetimeScope LifetimeScope,
-    BlockchainProcessor BlockchainProcessor,
+    BlockchainProcessor BlockchainProcessorImpl,
```

Comment:
Can we change it to?

```suggestion
    IBlockchainProcessor BlockchainProcessor,
```

Reply:
Not in this case I'm afraid, as `IBlockchainProcessor` does not implement `IProcessingQueue`.